src/cow-react/modules/swap/containers/EthFlowStepper/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  EthFlowStepper as Pure,
+  EthFlowStepperProps,
+  SmartOrderStatus,
+} from '@cow/modules/swap/pure/EthFlow/EthFlowStepper'
+import { useDetectNativeToken } from '@cow/modules/swap/hooks/useDetectNativeToken'
+import { Order } from 'state/orders/actions'
+import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
+
+type Props = {
+  storeOrder: Order | undefined
+}
+
+export function EthFlowStepper(props: Props) {
+  const { storeOrder } = props
+  const { native } = useDetectNativeToken()
+
+  const state = mapOrderToEthFlowStepperState(storeOrder)
+  const isEthFlowOrder = getIsEthFlowOrder(storeOrder)
+
+  if (!storeOrder || !state || !isEthFlowOrder) {
+    return null
+  }
+
+  const stepperProps: EthFlowStepperProps = {
+    nativeTokenSymbol: native.symbol as string,
+    tokenLabel: storeOrder.outputToken.symbol as string,
```

Comment:
what if there's no symbol?

Reply:
Good question! Will default to name, then address

Reply:
Using good old https://github.com/cowprotocol/cow-js/blob/fceb94bb888665513bd35d8c40cdabf0a749cd01/src/utils/format.ts#L424

Reply:
cool